lib/valkyrie/persistence/fedora/persister/orm_converter.rb

```diff
@@ -20,6 +26,16 @@ def attributes
                          .merge(id: id, new_record: false)
       end
 
+      # Get Fedora's lastModified value from the LDP response
+      def populate_native_lock(resource)
+        return unless @optimistic_locking_enabled
+        lastmod = object.response_graph.first_object([nil, RDF::URI("http://fedora.info/definitions/v4/repository#lastModified"), nil])
+        return unless lastmod
+
+        token = Valkyrie::Persistence::OptimisticLockToken.new(adapter_id: "native-#{adapter.id}", token: DateTime.parse(lastmod.to_s).httpdate)
+        resource.optimistic_lock_token << token
```

Comment:
Access the property via its constant, `Valkyrie::Persistence::Attributes::OPTIMISTIC_LOCK`